packages/atomic/src/components/lazy-index.ts

```diff
@@ -1,5 +1,15 @@
+import CommerceComponentMap from './commerce/lazy-index.js';
+import InsightComponentMap from './insight/lazy-index.js';
+import IpxComponentMap from './ipx/lazy-index.js';
+import RecommendationsComponentMap from './recommendations/lazy-index.js';
 import SearchComponentMap from './search/lazy-index.js';
 
-export default {...SearchComponentMap};
+export default {
+  ...SearchComponentMap,
+  ...CommerceComponentMap,
+  ...InsightComponentMap,
+  ...IpxComponentMap,
+  ...RecommendationsComponentMap,
```

Comment:
Reverse the order; we want the most important map to be spread last so that in the off chance we mess up, the damage is limited